admin_solutions/authentication.adoc

```diff
@@ -5,10 +5,308 @@ toc::[]
+
+[WARNING]
+====
+*[NEEDINFO]*: Is this troubleshooting information? It was in introduction of the install guide authentication topic, but seems out of place there. What purpose does this serve? Does any of it belong in the overview here, in other words does any of this need to be considered before configuring authentication?
+====
+
+A `401` response indicates failed authentication.
+
+A non-`200` status, or the presence of a non-empty "error" key, indicates an
+error:
+
+----
+{"error":"Error message"}
+----
+
+A `200` status with a `sub` (subject) key indicates success:
+
+----
+{"sub":"userid"} <1>
+----
+<1> The subject must be unique to the authenticated user and must not be able to
+be modified.
+
+A successful response may optionally provide additional data, such as:
+
+* A display name using the `name` key. For example:
++
+----
+{"sub":"userid", "name": "User Name", ...}
+----
++
+* An email address using the `email` key. For example:
++
+----
+{"sub":"userid", "email":"user@example.com", ...}
+----
++
+* A preferred user name using the `preferred_username` key. This is useful when
+the unique, unchangeable subject is a database key or UID, and a more
+human-readable name exists. This is used as a hint when provisioning the
+OpenShift user for the authenticated identity. For example:
++
+----
+{"sub":"014fbff9a07c", "preferred_username":"bob", ...}
+----
+
+[[create-users-basic-auth]]
+=== Creating Users with Basic Authentication
+
+You do not create users in {product-title} when integrating with an external
+authentication provider, such as, in this case, Basic authentication. The Basic
+authentication server is the system of record, meaning that users are defined
+there, and any user with a valid user name for the configured authentication
+server can log in.
+
+To add a user to {product-title}, the user must exist on the Basic
+authentication server, and if required you must create a new Basic
+authentication account for the user.
+
+[WARNING]
+====
+*[NEEDINFO]*: After reading the wikipedia page on Basic access authentication, something tells me that the above section isn't true for this particular authentication method.
+====
+
+[[view-users-basic-auth]]
+=== Viewing Users
+
+Once one or more users have logged in, you can run `oc get users` to view a
+list of users and verify that users were created successfully:
+
+.Output of `oc get users` command
+====
+
+----
+$ oc get users
+NAME         UID                                    FULL NAME   IDENTITIES
+bobsmith     a0c1d95c-1cb5-11e6-a04a-002186a28631   Bob Smith   basicauth:bobsmith <1>
+----
+<1> Identities in {product-title} are comprised of the identity provider name prefixed to the Basic authentication username.
+====
+
+[WARNING]
+====
+*[NEEDINFO]*: Do you have access to a basic authentication setup? If so, can you please provide me with the output for the `oc get users` command when basic auth is configured as the auth provider for OpenShift? Thank you.
```

Comment:
this step doesn't make sense to repeat for every identity provider... `oc get users` looks the same, no matter what IDP you use.